test/Microsoft.ML.TestFrameworkCommon/Attributes/NativeDependencyFact.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.TestFrameworkCommon.Utility;
+using Microsoft.ML.TestFrameworkCommon.Attributes;
+
+namespace Microsoft.ML.TestFramework.Attributes
+{
+    public sealed class NativeDependencyFact : EnvironmentSpecificFactAttribute
```

Comment:
Attribute class names should have `Attribute` suffixes. This applies to NativeDependencyTheory as well.

Reply:
Fixed.